### PR TITLE
[Fairground 🎡] Flexible special trail

### PR DESCRIPTION
**Pull request by abeddow91**
## What does this change?
Reworks the logic for supportingContent (sublinks) on cards.

The new logic dictates that we only display sublinks on an "inner" position on tablet breakpoint and above. Otherwise, they appear in the "outer" position which sits them below image and content unless there position is set to "none" in which case they do not appear at all.

## Why?
We do not show sublinks above images or content on breakpoints below tablet so this change codifies that design decision.

## Screenshots

| Wide      | Mobile      |
| ----------- | ---------- |
| ![before][] | ![after][] |

[before]: https://github.com/user-attachments/assets/e08a5dd9-7d9e-4eac-8b97-77f8b0981ff5
[after]: https://github.com/user-attachments/assets/c2cd7c9f-7f4b-4399-89c6-f148dea4fbcb

### CODE REVIEW

**Review by Georges-GNM — APPROVED** [review]
LGTM

File: dotcom-rendering/src/components/Card/Card.tsx [hunk 1]
```diff
@@ -376,6 +376,38 @@ export const Card = ({
 		return 'medium';
 	};
 
+	/**
+	 * Determines how and when to render the `SupportingContent` component in the "outer" position:
+	 * - Returns `null` if `supportingContent` is unavailable or `sublinkPosition` is `none`.
+	 * - Renders `SupportingContent` for all breakpoints if `sublinkPosition` is `outer`.
+	 * - If `sublinkPosition` is `inner`, hides `SupportingContent` on desktop but displays it on smaller breakpoints.
```
Comment (abeddow91) [inline dotcom-rendering/src/components/Card/Card.tsx:383]:
```suggestion
	 * - If `sublinkPosition` is `inner`, hides `SupportingContent` from tablet but displays it on smaller breakpoints.
```

File: dotcom-rendering/src/components/Card/Card.tsx [hunk 2]
```diff
@@ -376,6 +376,38 @@ export const Card = ({
...
 	};
 
+	/**
+	 * Determines how and when to render the `SupportingContent` component in the "outer" position:
+	 * - Returns `null` if `supportingContent` is unavailable or `sublinkPosition` is `none`.
+	 * - Renders `SupportingContent` for all breakpoints if `sublinkPosition` is `outer`.
+	 * - If `sublinkPosition` is `inner`, hides `SupportingContent` from tablet but displays it on smaller breakpoints.
+	 *
+	 */
+	const decideOuterSublinks = () => {
```
Comment (Georges-GNM) [inline dotcom-rendering/src/components/Card/Card.tsx:386]: Nice to have extracted this logic!

**Review by Georges-GNM — APPROVED** [review]
👏

**Pull request merged**